library/src/main/java/com/datatorrent/lib/io/fs/FileSplitter.java

```diff
@@ -79,7 +79,7 @@
  */
 @OperatorAnnotation(checkpointableWithinAppWindow = false)
 @Deprecated
-public class FileSplitter implements InputOperator, Operator.CheckpointListener
+public class FileSplitter implements InputOperator, Operator.CheckpointListener, Operator.CheckpointNotificationListener
```

Comment:
Is this necessary for backward compatibility of just because of japicmp? If the latter, can the change be whitelisted?